### PR TITLE
fix exporter values.yaml identation

### PR DESCRIPTION
**Pull request by dioguerra**
Signed-off-by: Diogo Guerra <diogo.filipe.tomas.guerra@cern.ch>

### CODE REVIEW

**Comment by zyyw** [comment]
Hi @dioguerra , could you please resolve the conflicts? Thanks for your effort!

**Review by ywk253100 — APPROVED** [review]

**Comment by dioguerra** [comment]
@zyyw done.

**Review by zyyw — APPROVED** [review]

File: values.yaml [hunk 1]
```diff
@@ -820,25 +820,23 @@ redis:
...
-    cacheDuration: 23
-    cacheCleanInterval: 14400
-    ## The priority class to run the pod as
-    priorityClassName:
+  #   requests:
+  #     memory: 256Mi
+  #     cpu: 100m
+  podAnnotations: {}
+  serviceAccountName: ""
+  image:
```
Comment (zyyw) [inline values.yaml:830]: could you please justify why this PR want to remove these two lines:
```
    # mount the service account token
    automountServiceAccountToken: false
```

**Review by zyyw — COMMENTED** [review]
please reply the question of my previous comment. thanks

**Comment by zyyw** [comment]
@dioguerra  could you please update the PR per the comments above? thanks

File: values.yaml (hunk 1 above)
Comment (dioguerra) [reply values.yaml:830]: Apologies if i droped this. I will fix it again, when i can

**Comment by zyyw** [comment]
@dioguerra it look like this indentation is fix in this PR (merged recently): https://github.com/goharbor/harbor-helm/pull/1033

I'm going to close this PR. But we do appreciate your effort on fixing the indentation issue. Thanks!

**Pull request closed without merging**